homeassistant/components/openrgb/config_flow.py

```diff
@@ -2,16 +2,21 @@
 
 from __future__ import annotations
 
+from functools import partial
+from ipaddress import IPv6Address, ip_address
 import logging
 from typing import Any
 
+from getmac import get_mac_address
```

Comment:
We shouldn't use getmac. getmac uses the ARP table, and in the past this was troublesome with people with advanced network, which for example caused wrong mac can be found (like for example the managed switch in between) and then everything goes south. Otherwise way more integrations would use getmac :)

Reply:
What if I ensure the IP change only gets applied if a successful connection to OpenRGB can be established with the new IP?

That would mean inferring an incorrect MAC would be harmless.

Reply:
I opened a discussion upstream:

- https://gitlab.com/CalcProgrammer1/OpenRGB/-/issues/5244